shared/teams/team/header/index.tsx

```diff
@@ -109,7 +109,6 @@ const _TeamHeader = (props: Props) => (
               type="iconfont-chat"
               style={Kb.iconCastPlatformStyles(styles.chatIcon)}
               color={Styles.globalColors.white}
-              size={16}
```

Comment:
Just checking this is intentional

Reply:
as far as I can tell, it isn't used. In other places I kept it for now. But I'll make sure to test this and maybe remove the other cases as well